ir.c

```diff
@@ -325,7 +329,7 @@ void ir_init(ir_ctx *ctx, ir_ref consts_limit, ir_ref insns_limit)
 
 	ctx->strtab.data = NULL;
 
-	buf = ir_mem_malloc((consts_limit + insns_limit) * sizeof(ir_insn));
+	buf = ir_mem_calloc(consts_limit + insns_limit, sizeof(ir_insn));
```

Comment:
calloc() may be more expensive as it needs to clear the memory

Reply:
Yeah, `calloc` is definitely more expensive. It was in an early version of the patch where it seemed some members of `ctx->ir_base` were not initialized to zero and that might have caused some failures. I've reverted this and seems it doesn't reproduce the issues i've initially seen. So keep it reverted. 

Thanks